### PR TITLE
Add September newsletter

### LINKED ISSUE #327: September newsletter
Fixes https://github.com/readthedocs/website/issues/326

### PR DESCRIPTION
**Pull request by ericholscher**
Closes #327 

----
📚 Documentation preview 📚: https://readthedocs-about--326.org.readthedocs.build/

### CODE REVIEW

File: content/posts/newsletter-september-2024.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* We continue to improve the UX of importing projects, and now check to see if you have a `.readthedocs.yml` file in your repository, and suggest a starting point if not.
+* We've added Multi-Factor Authentication (MFA) to our platform, which is currently in beta testing. It will be fully rolled out in the next month or two as we work a couple kinks out.
+* Our [API v3](https://docs.readthedocs.io/en/stable/api/v3.html) now supports anonymous requests for many endpoints, which should make it easier to integrate in documentation pages, and other use cases.
+
+You can always see the latest changes to our platforms in our [changelog 📃](https://docs.readthedocs.io/page/changelog.html).
+
+## Upcoming changes
+
+* We're working on a project to make it [easier to understand what files have changed between versions](https://github.com/readthedocs/readthedocs.org/issues/11319), which will allow us to build a number of neat features like Redirect recommendations, and Pull Request page review suggestions.
```
Comment (humitos) [inline content/posts/newsletter-september-2024.md:27]:
```suggestion
* We're working on a project to make it [easier to understand what files have changed between versions](https://github.com/readthedocs/readthedocs.org/issues/11319), which will allow us to build a number of neat features like redirect recommendations, and pull request page review suggestions.
```

File: content/posts/newsletter-september-2024.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+date: September 13, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/september-2024.jpg
+image_credit: Photo by <a href="https://unsplash.com/@routessansfins?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Les routes sans fin(s)</a> on <a href="https://unsplash.com/photos/a-lifeguard-tower-sitting-on-top-of-a-sandy-beach-gnkhKUmENms?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+## News and updates
```
Comment (humitos) [inline content/posts/newsletter-september-2024.md:11]: Do we want to mention SAML in beta testing again so we can get more customers here and promote it a little more?

File: content/posts/newsletter-september-2024.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* We've added Multi-Factor Authentication (MFA) to our platform, which is currently in beta testing. It will be fully rolled out in the next month or two as we work a couple kinks out.
+* Our [API v3](https://docs.readthedocs.io/en/stable/api/v3.html) now supports anonymous requests for many endpoints, which should make it easier to integrate in documentation pages, and other use cases.
+
+You can always see the latest changes to our platforms in our [changelog 📃](https://docs.readthedocs.io/page/changelog.html).
+
+## Upcoming changes
+
+* We're working on a project to make it [easier to understand what files have changed between versions](https://github.com/readthedocs/readthedocs.org/issues/11319), which will allow us to build a number of neat features like Redirect recommendations, and Pull Request page review suggestions.
+* We are looking at adding functionality similar to our [sphinx-hoverxref extension](https://github.com/readthedocs/sphinx-hoverxref) to our new documentation system, which support popups for cross-references in a single documentation page. This will work for all documentation tools, not just Sphinx.
```
Comment (humitos) [inline content/posts/newsletter-september-2024.md:28]: Shouldn't we call it "Read the Docs Addons" instead of "new documentation system"?

**Review by humitos — APPROVED** [review]

File: content/posts/newsletter-september-2024.md (hunk 3 above)
Comment (ericholscher) [reply content/posts/newsletter-september-2024.md:28]: I just copied this from last month 😆

File: content/posts/newsletter-september-2024.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+title: Read the Docs Newsletter - September 2024
+date: September 13, 2024
```
Comment (ericholscher) [inline content/posts/newsletter-september-2024.md:2]:
```suggestion
date: September 10, 2024
```

**Pull request merged**